components/lib/crash/src/main/java/mozilla/components/lib/crash/prompt/CrashReporterActivity.kt

```diff
@@ -85,4 +93,13 @@ class CrashReporterActivity : AppCompatActivity() {
             }
         }
     }
+
+    override fun onStop() {
+        super.onStop()
+
+        /* Save the state of send checkbox into shared preferences. We can't use onSaveInstanceState here
+            because the state of the checkbox should also be preserved upon app restart.
+        */
```

Comment:
Mentioning `onSaveInstanceState` may be confusing here. Just some short comment that we are remembering this for the next time the app crashes should be enough :)